phpBB/styles/prosilver/theme/content.css

```diff
@@ -466,17 +466,24 @@ blockquote cite {
 	/* Username/source of quoter */
 	font-style: normal;
 	font-weight: bold;
-	margin-left: 20px;
+	margin-left: 0;
 	display: block;
 	font-size: 0.9em;
 }
 
+blockquote cite:before, .uncited:before {
+	/* Font Awesome quote-left */
+	content: '\f10d';
```

Comment:
this should probably be moved to the icon.css file so all icons are in the same place.

also we should match the icon class with this for consistency in rendering

```css
blockquote cite:before,
.uncited:before {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
	font: normal normal normal 14px/1 FontAwesome;
   	font-size: inherit;
    text-rendering: auto;
    display: inline-block;
    padding-right: 5px;
	content: "\f10d"; /* Font Awesome quote-left */
}
```

this may require tweaking some of the other css but it will help to ensure all the icons render the same